dist/win/resources/debug-launcher.properties

```diff
@@ -0,0 +1,18 @@
+win-console=true
+java-options=--enable-preview \
+             --enable-native-access=org.cryptomator.jfuse.win \
+             -Xss5m \
+             -Xmx256m \
+             -Dcryptomator.appVersion="${SEM_VER_STR}" \
+             -Dfile.encoding="utf-8" \
+             -Dcryptomator.logDir="~/AppData/Roaming/$AppName" \
+             -Dcryptomator.pluginDir="~/AppData/Roaming/$AppName/Plugins" \
+             -Dcryptomator.settingsPath="~/AppData/Roaming/$AppName/settings.json" \
+             -Dcryptomator.ipcSocketPath="~/AppData/Roaming/$AppName/ipc.socket" \
+             -Dcryptomator.p12Path="~/AppData/Roaming/$AppName/key.p12" \
+             -Dcryptomator.mountPointsDir="~/$AppName" \
+             -Dcryptomator.loopbackAlias="$LoopbackAlias" \
+             -Dcryptomator.integrationsWin.autoStartShellLinkName="$AppName" \
+             -Dcryptomator.integrationsWin.keychainPaths="~/AppData/Roaming/$AppName/keychain.json" \
+             -Dcryptomator.showTrayIcon=true \
+             -Dcryptomator.buildNumber="msi-${REVISION_NUM}"
```

Comment:
We might even add `-dbg` here, what do you think?

Reply:
You mean a new jvm property (e.g.,  `cryptomator.dbg`) which is read by the application?

Reply:
No just the build number, so we know whether people started the debug launcher

Reply:
Ah, got it! Yes, that's a good idea.